app/models/miq_queue_worker_base/runner.rb

```diff
@@ -104,6 +104,7 @@ def deliver_queue_message(msg)
 
     begin
       $_miq_worker_current_msg = msg
+      Thread.current["request_id"] = msg.task_id || msg.request_uuid
```

Comment:
Should we change MiqQueue.put to automatically default request_uuid to task_id?  That way this code can just be

```ruby
Thread.current["request_id"] = msg.request_uuid
```